incubator/quarkus/templates/kafka/docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: '2'
+
+services:
+
+  zookeeper:
+    image: strimzi/kafka:0.15.0-kafka-2.3.1
+    command: [
+      "sh", "-c",
+      "bin/zookeeper-server-start.sh config/zookeeper.properties"
+    ]
+    ports:
+      - "2181:2181"
+    environment:
+      LOG_DIR: /tmp/logs
+
+  kafka:
+    image: strimzi/kafka:0.15.0-kafka-2.3.1
+    command: [
+      "sh", "-c",
+      "bin/kafka-server-start.sh config/server.properties --override listeners=$${KAFKA_LISTENERS} --override advertised.listeners=$${KAFKA_ADVERTISED_LISTENERS} --override zookeeper.connect=$${KAFKA_ZOOKEEPER_CONNECT}"
+    ]
+    depends_on:
+      - zookeeper
+    ports:
+      - "9092:9092"
+    environment:
+      LOG_DIR: "/tmp/logs"
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092
```

Comment:
I see `KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://localhost:9092` in the original QS. Is this hostname change needed to make it work with Appsody?

Reply:
Yes it's needed so that the `appsody run` container can talk to Kafka over the Docker network that this `docker-compose.yaml` creates.  We've used the same approach in other stacks (Spring, Node.js etc.)

Reply:
OK, good to know. Thanks.